Assignment_2.md

```diff
@@ -0,0 +1,145 @@
+package Core_Java_Assignments;
+
+import java.util.Scanner;
+
+class calculator {
```

Comment:
Class name should be in the upper camel case

Reply:
yes sir